src/app/app-layout/header/header.component.html

```diff
@@ -1,14 +1,17 @@
 <div class="bg">
   <img src="assets/img/logo.png" srcset="assets/img/logo@2x.png 2x,
              assets/img/logo@3x.png 3x" class="logo">
+  <div *ngIf="userInfo$ | async" class="user-info">
+    Welcome&nbsp;{{userInfo.name.first}}&nbsp;{{userInfo.name.last}}!
```

Comment:
We could simply use the regular space character... Why did you use `&nbsp;`

Reply:
Fixed. Thanks for catching this!